### PR TITLE
fix: fix broken build with cuda

### PR DESCRIPTION
**Pull request by chokobole**
# Description

This PR fixes broken build with cuda

### CODE REVIEW

**Review by ashjeong — APPROVED** [review]
LGTM

**Review by dongchangYoo — APPROVED** [review]
LGTM

File: tachyon/base/parallelize_unittest.cc [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #include "tachyon/base/parallelize.h"
 
 #include <functional>
+#include <numeric>
```
Comment (TomTaehoonKim) [inline tachyon/base/parallelize_unittest.cc:4]: Is this for std::accumulate?

**Review by TomTaehoonKim — APPROVED** [review]
Else, LGTM

**Review by Insun35 — APPROVED** [review]
LGTM

File: tachyon/base/parallelize_unittest.cc (hunk 1 above)
Comment (chokobole) [reply tachyon/base/parallelize_unittest.cc:4]: Yeap

**Pull request merged**